src/client/components/features/OffreDeStage/Consulter/ConsulterOffreDeStage.tsx

```diff
@@ -23,10 +23,17 @@ export function ConsulterOffreDeStage({ offreDeStage }: ConsulterOffreDeStagePro
 		];
 	}, [offreDeStage]);
 
-	const descriptionEmployeurHtmlSanitiezd =  useSanitize(offreDeStage.employeur.description ? getHtmlFromMd(offreDeStage.employeur.description) : undefined);
+	const descriptionEmployeurHtmlSanitiezd = useSanitize(offreDeStage.employeur.description ? getHtmlFromMd(offreDeStage.employeur.description) : undefined);
 	const descriptionHtmlSanitized = useSanitize(getHtmlFromMd(offreDeStage.description));
 
-	const salaireOffreDeStage = offreDeStage.remunerationBase?.toString();
+	function getRemunerationOffreDeStage() {
+		if (offreDeStage.remunerationBase === undefined) {
+			return 'Non renseignée';
```

Comment:
comment c'est possible que le test passe avec renseigné sans ée ?

Reply:
to haveTextContent ça regarde si la portion que tu lui donnes est présente pas l'entièreté